.github/workflows/merge.yml

```diff
@@ -199,14 +215,25 @@ jobs:
         with:
           command: |
             mvn verify -B -Daudit \
-              -pl "${{ inputs.modules_to_build || steps.change_detection.outputs.changed_modules }}" -amd
+              -pl "${{ inputs.modules_to_build || format('{0},{1}', inputs.modules_to_always_build, steps.change_detection.outputs.changed_modules) }}" \
+              -amd
         env:
           cmd_type: UNIT_TEST
           reporter: 'java-junit'
           fail-on-error: 'false'
           GITHUB_TOKEN: ${{ github.token }}
```

Comment:
We have updated our action. This env variable `GITHUB_TOKEN` is not required now.

Reply:
Removed